addons/web_editor/controllers/main.py

```diff
@@ -618,7 +621,8 @@ def save_library_media(self, media):
                 'res_id': 0,
             })
             if media[id]['is_dynamic_svg']:
-                attachment['url'] = '/web_editor/shape/illustration/%s' % slug(attachment)
+                colorParams = werkzeug.urls.url_encode({'c%s' % color_name: color for color_name, color in media[id]['dynamic_colors'].items()})
```

Comment:
is there a good reason to reprepare dict here instead to use c<color> directly here: https://github.com/odoo/odoo/pull/60503/files#diff-de463c6eb35f75c6d30e3756ffedf0d613195891c9f92ed7d023e3b16e07bd9cR1041